.buildkite/scripts/release-manager.sh

```diff
@@ -21,8 +21,8 @@ readonly TYPE=${TYPE:-snapshot}
 readonly OUTPUT_FILE=${OUTPUT_FILE:-release-manager-report.out}
 
 # set required permissions on artifacts and directory
-chmod -R a+r "$FOLDER"/*
-chmod -R a+w "$FOLDER"
+chmod -R a+r "${WORKSPACE}"/"$FOLDER"/*
+chmod -R a+w "${WORKSPACE}"/"$FOLDER"
```

Comment:
Would it be the same if 
FOLDER would get that value from dra_release.sh ?
Suggestion added in the other comment

To keep the script as it was (if possible)